hedera-node/hedera-app/src/main/java/com/hedera/node/app/config/BootstrapConfigProviderImpl.java

```diff
@@ -57,11 +59,13 @@ public BootstrapConfigProviderImpl() {
                 .withSource(SystemEnvironmentConfigSource.getInstance())
                 .withSource(SystemPropertiesConfigSource.getInstance())
                 .withSource(new PropertyConfigSource(SEMANTIC_VERSION_PROPERTIES_DEFAULT_PATH, 500))
+                .withConfigDataType(BlockStreamConfig.class)
                 .withConfigDataType(BootstrapConfig.class)
                 .withConfigDataType(HederaConfig.class)
                 .withConfigDataType(FilesConfig.class)
                 .withConfigDataType(VersionConfig.class)
                 .withConfigDataType(LedgerConfig.class)
+                .withConfigDataType(AccountsConfig.class)
```

Comment:
Intentional? I don't see anywhere else in this PR that this config is used

Reply:
Just took changes from spike branch. I think that config was missing also in the bootstrap config and a unit test was failing.